app/views/students/index.html.haml

```diff
@@ -1,11 +1,3 @@
-%h1 Students
-
-%ul
-  - @students.each do |student|
-    .student
-      %p.student-name= link_to student.name, student_path(student)
-
-%hr
-
-%h5 Add New Student
-%div= render 'form'
+.row
+  %div#left_nav_index{ class: 'small-2 columns' }= render 'left_nav_index'
```

Comment:
`%div#left_nav_index` can be written as `#left_nav_index` since `%div` is implicit<br>Avoid defining `class` in attributes hash for static class names